test/commands/git/sync_concept_exercise_test.rb

```diff
@@ -11,7 +11,7 @@ class Git::SyncConceptExerciseTest < ActiveSupport::TestCase
   end
 
   test "git SHA does not change when there are no changes" do
-    head_sha = "73b8e389daac4a3e08747e6c16fd420072386e2b"
+    head_sha = "7a42b345ebfa4e74899174abb4ed6b9301cf93b9"
```

Comment:
Can we look this up from git, rather than having to change it each time?

Reply:
Yeah we can. Fixed

Reply:
Great. Thanks :)